src/helpers/helpers.core.js

```diff
@@ -306,6 +306,18 @@ var helpers = {
 
 		ChartElement.__super__ = me.prototype;
 		return ChartElement;
+	},
+
+	/**
+	 * If argument is string, return it. If it is an array, join its elements using
+	 * a space string (' ').
+	 * @private
+	 */
+	_joinIfArray: function(stringOrArray) {
```

Comment:
This method is only used in `plugin.legend.js`, let's move it [there](/chartjs/Chart.js/blob/a92bc26a5f686d547ef14c9e2fd5e838769813c3/src/plugins/plugin.legend.js#L98).

Reply:
Done